diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/grace/wdi/try/
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+GitHub is a respository (a folder in the cloud), that saves and keep trakes of your files. It's considerd a source code management system which works on top of Git.
```

Comment:
Correct. Also think of it as a User Interface for us to see Git and Version Control on our repositories.